src/ui/ui_settings/__tests__/index.js

```diff
@@ -5,6 +5,18 @@ import { uiSettingsMixin } from '../ui_settings_mixin';
 import { getDefaultSettings } from '../defaults';
 import { errors as esErrors } from 'elasticsearch';
 
+async function expectRejection(promise, errorMessageContain) {
```

Comment:
Would be nice to have Jest: https://facebook.github.io/jest/docs/en/asynchronous.html#resolves-rejects ;)